test/celeritas/optical/Generator.test.cc

```diff
@@ -241,8 +241,8 @@ TEST_F(LArSphereGeneratorTest, generator)
     if (reference_configuration)
     {
         EXPECT_EQ(51226, gen.num_generated);
-        EXPECT_EQ(54319, result.steps);
-        EXPECT_EQ(15, result.step_iters);
+        EXPECT_EQ(53429, result.steps);
+        EXPECT_EQ(14, result.step_iters);
```

Comment:
This shouldn't have changed since the lar sphere doesn't use a scintillation spectrum, it uses gaussian. Right?